docs/source/tutorials/datasets.rst

```diff
@@ -52,15 +52,15 @@ function. You can see a list of all supported datasets :ref:`here<datasets>`.
     tune run full_finetune_single_device --config llama3/8B_full_single_device \
     dataset=torchtune.datasets.alpaca_dataset
 
-Hugging Face datasets
----------------------
+Hugging Face and Local datasets
+-------------------------------
 
 We provide first class support for datasets on the Hugging Face hub. Under the hood,
 all of our built-in datasets and dataset builders are using Hugging Face's ``load_dataset()``
 to load in your data, whether local or on the hub.
 
 You can pass in a Hugging Face dataset path to the ``source`` parameter in any of our builders
-to specify which dataset on the hub to download. Additionally, all builders accept
+to specify which dataset on the hub to download or use from a local directory path. Additionally, all builders accept
```

Comment:
could you add a ref link to the local datasets section here? Since source has to be configured slightly differently when using local files. See [here](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#implicit-targets) for how to do that with RST

Reply:
I am not sure to understand where the ref link is supposed to point to. Can you be more explicit ?

Reply:
https://pytorch.org/torchtune/stable/tutorials/datasets.html#local-and-remote-datasets

Reply:
sorry, I meant to add it like this:
```suggestion
to specify which dataset on the hub to download or use from a local directory path (see `Local and remote datasets`_). Additionally, all builders accept
```